modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -0,0 +1,48 @@
+= Node-wise Partition Recovery
+:description: Feature to recover partitions that have lost a majority of replicas.
+
+Multi-node or entire-AZ failures (especially in cloud environments) and some forms of human error may result in ‘stuck’ partitions with fewer replicas than required to make a quorum. In these scenarios, some data loss is unavoidable but admins need to recover what they can and move on.
+
+IMPORTANT: This is a last ditch measure when all other recovery options have failed. In some cases, no remaining replicas may exist for the partitions on the dead nodes. This recovery method is used in scenarios where you are already experiencing data loss - the goal is to stop the loss of any more data.
+
+== Assumptions
+
+This capability only attempts to recover data from partitions where the following conditions are met:
+
+* Your cluster must have one or more irrecoverable nodes.
+* One or more partitions on these nodes must have no majority.
+* Some of the data on these partitions is no longer recoverable.
```

Comment:
@bharathv is this last part really accurate?  For some of the partitions, you will still have the leader and NO data will be unrecoverable.   let's say in the future we have leader pinning and there are no leaders at all on the nodes you're recovering from.  You may have stuck partitions due to lost majority but may end up with all the data at the end.

Reply:
@greg-szrama 

IMO the way to explain this feature  is that it is intended for situations when partitions have lost majority for whatever reason, including irrecoverable nodes or user error.  

We say that up front (that it could be just human error). but then we say 'only where the following conditions are met', and not all of those 3 things must be true (e.g. in the case of human error the nodes might be lost, in the case of only follower replicas on the target nodes, there's no unrecoverable data from those partitions).

So I don't think the assumptions section is really helpful here. IMO we're overcomplicating the story. 

The command reviews all partitions from those nodes, detects any partitions on them that have lost majority and therefore may have unrecoverable data.

We also don't explain that we move all replicas OFF the target nodes in the process. So we are both repairing the partitions and also draining the target nodes of all their replicas in one step.

Reply:
I removed the section and added a paragraph to explain your last note. Let me know your thoughts.